### ISSUE TITLE
Sphinx multiline return broken.

### ISSUE THREAD

**Issue by terrencepreilly** [terrencepreilly/darglint#63]
Having a multiline return in sphinx is broken: any line breaks result in no return being parsed.

See https://github.com/terrencepreilly/darglint/issues/56#issuecomment-588276997 for an example.

**Comment by lvermue** [comment]
@terrencepreilly Thank you for your quick response.
In addition to the return there appears to be the same problem with raises, e.g.
```
:raises RuntimeError:
    Long description of why this happens.
```

does not work and raises _DAR401_.

**Comment by terrencepreilly** [comment]
This should be resolved in v1.2.1.
